CMakeLists.txt

```diff
@@ -360,8 +358,18 @@ add_library(wabt STATIC ${WABT_LIBRARY_SRC})
 
 IF (NOT WIN32)
   add_library(wasm-rt-impl STATIC wasm2c/wasm-rt-impl.c wasm2c/wasm-rt-impl.h)
-  install(TARGETS wasm-rt-impl DESTINATION ${CMAKE_INSTALL_LIBDIR})
-  install(FILES wasm2c/wasm-rt.h wasm2c/wasm-rt-impl.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+  if (WABT_INSTALL_RULES)
+    install(
+      TARGETS wasm-rt-impl
+      COMPONENT WABT_Development
+    )
```

Comment:
Does this not fit on one line?

Reply:
Is there a naming convention that cmake tests to use for components?  My inclination would be use all lowercase here (e.g. wabt_development) because we don't use the uppercase name wabt in source code or filenames.

Reply:
Why don't we need `DESTINATION` here anymore?  Previously we had it set to `CMAKE_INSTALL_LIBDIR`... is that the default?

Reply:
It does fit on one line. Happy to do that, was just trying to be consistent. There is not an established convention, I was using the `<ProjectName>_Component` pattern that Craig Scott, one of the maintainers, [advocated in his CppCon talk](https://www.youtube.com/watch?v=m0DwB4OvDXk). Happy to make it `wabt-development` if you prefer.

Reply:
`DESTINATION` is no longer needed because it takes the defaults from `GNUInstallDirs`, `CMAKE_INSTALL_LIBDIR` included.

Reply:
Done.